### DISCUSSION TITLE
4.0.5When I delete,the file on the disk still exists only when it is deleted from the download list.

### DISCUSSION THREAD

**Discussion by tianyi725** [transmission/transmission#8491] [Support]
### What is the issue?

I installed version 4.0.5 the way I did on github.

When I delete, I select "trash data and remove from list..."

However, the file on the disk still exists only when it is deleted from the download list.

What is the reason for this?

### Which application of Transmission?

transmission-daemon

### Which version of Transmission?

4.0.5

**Comment by killemov** [comment]
Autotranslate?

This happens when the disk is full.

**Comment by jancoow** [comment]
I've got the same issue. Disk is not full. No errors in the log.
